index.html

```diff
@@ -1,14 +1,161 @@
+          <div class="date">2020.10.15(THU) ~ 16(FRI)</div>
+          <footer class="national">@ National Museum of Korea, Art Center Nabi and more</footer>
+        </section>
+      </div>
+      <section class="program">
+        <header>
+          <div class="text">Main Program</div>
+          <div class="horizontal-rule"></div>
+        </header>
+        <ul class="elements">
+          <li class="element">
+            <img class="icon" src="images/icons/program_lecture.svg" ></img>
+            <div class="title">Lecture</div>
+            <h5 class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Impedit id rerum ex neque non.</h5>
+          </li>
+          <li class="element">
+            <img class="icon" src="images/icons/program_cc_exhibition.svg"></img>
+            <div class="title">CC Exhibition</div>
+            <h5 class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Minima ipsa ad mollitia atque?</h5>
+          </li>
+          <li class="element">
+            <img class="icon" src="images/icons/program_forum.svg"></img>
+            <div class="title">Forum</div>
+            <h5 class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam omnis ex consequatur hic cumque!</h5>
+          </li>
+          <li class="element">
+            <img class="icon" src="images/icons/program_workshop.svg"></img>
+            <div class="title">Workshop</div>
+            <h5 class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloremque corporis aut </h5>
+          </li>
+          <li class="element">
+            <img class="icon" src="images/icons/program_cc_ignite.svg"></img>
+            <div class="title">CC Ignite</div>
+            <h5 class="description">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolorem dolore ut itaque cum distinctio!</h5>
+          </li>
+        </ul>
+        <footer class="join">Join Free Thinking 2020</footer>        
+        <footer class="join-desk"><a href="">SEE THE WHOLE PROGRAM</a></footer>
+      </section>
+      <section class="speakers">
+        <div class="title-6">Featured Speakers</div>        
+        <div class="horizontal-rule"></div>
+        <div class="speaker-container">
+        <!--  in this container all the speakers will be added dinamically -->
+        </div>
+        <div class="more">
+          <div class="text">MORE</div>
+          <img class="down_arrow" src="images/icons/down_arrow.svg" alt="down arrow">
+        </div>
+      </section>
+
+    </div>
+    <!-- ////////////////////////////////////////////////////////////////////  -->
+    <!-- /                          SECOND PAGE                              / -->
+    <!-- ////////////////////////////////////////////////////////////////////  -->
+    <div class="about" id="about">      
+      <section class="hero-1">
```

Comment:
- [ ] Dear @danifromecuador, I  notice you have organized your project in a single-page manner, but for this project, we need to make sure the project contains a Home page and an About page. Please make sure to make these two sections separate pages having a navigation bar and footer of their own and It's possible to navigate between the pages.
 - [ ] Please, also remember both pages have a different layout when it comes to the footer.  Consider this when you are working on the separate page feature of the two pages.

![image](https://user-images.githubusercontent.com/69077061/226001082-f917d2b9-f2ee-4450-94e3-162cf66667ad.png)
- [ ] Please also notice, the footer for the about page on the desktop size is different from the one on a mobile display. Please notice this too before the next review request. 
### About page footer on desktop 
![image](https://user-images.githubusercontent.com/69077061/226002106-1fea10f9-1533-4180-b997-e47b3d546865.png)
### About page footer on mobile 
![image](https://user-images.githubusercontent.com/69077061/226002524-c4f3ece0-ee2a-4111-8800-06b3347c3b62.png)